app/auth/views/logout.py

```diff
@@ -8,4 +8,9 @@
 def logout():
     logout_user()
     flash("You are logged out", "success")
-    return redirect(url_for("auth.login"))
+    response = make_response(redirect(url_for("auth.login")))
+    response.delete_cookie("slapp")
```

Comment:
I wonder if we can use `session.pop` here to make sure to remove all cookies except the `_flashes` which is used for `flash`?

https://stackoverflow.com/a/51395792/1428034

Reply:
The session contains server session variables. Cookies are associated with the request object.